bittensor/metagraph.py

```diff
@@ -70,7 +71,7 @@ def latest_block_path(dir_path: str) -> str:
         return latest_file_full_path
 
 
-class metagraph(torch.nn.Module):
+class metagraph:
```

Comment:
suggestion: since we are breaking API anyhow, maybe lets rename it to 
```suggestion
class Metagraph:
```
we can add 
`metagraph = Metagraph` alias at the bottom if we don't want to annoy people too much, but there likely is no reason to keep this class lowercase